src/oas/accessors.ts

```diff
@@ -21,9 +21,13 @@ export function createOasParamsIterator(
     const seenParams = new Set();
     const { parentId, context } = this;
     const opParams = Array.isArray(operation.parameters) ? operation.parameters : [];
-    const params = [...opParams, ...(Array.isArray(path.parameters) ? path.parameters : [])];
+    const pathParams = Array.isArray(path.parameters) ? path.parameters : [];
+    const params = [...opParams, ...pathParams];
 
     for (let i = 0; i < params.length; i++) {
+      this.context = i < opParams.length ? 'operation' : 'path';
+      this.parentId = this.ids[this.context];
+
```

Comment:
```suggestion
```
we could just remove it I think. It shouldn't be needed at all for now

Reply:
ah could be, I don't remember that code that much anymore even though I probably wrote it 😆